terraform/files/bin/clusterctl_template.sh

```diff
@@ -32,7 +32,7 @@ CLOUD_CONF_ENC=$(base64 -w 0 ~/$CLUSTER_NAME/cloud.conf)
 #Get CA and Encode CA
 # Update OPENSTACK_CLOUD
 if test "$OS_CLOUD" != "$OLD_OS_CLOUD"; then
-  echo "#Info: Changing OPENSTACK_CLOUD frpm $OLD_OS_CLOUD to $OS_CLOUD"
+  echo "#Info: Changing OPENSTACK_CLOUD from $OLD_OS_CLOUD to $OS_CLOUD"
```

Comment:
This change does not seem to be related to the actual change.

(Nothing critical of course, but I was assigned to review, so I review :D)

Reply:
You are right -- I was lazy and did the *cosmetic* fix here, while it should have been done in a separate branch.
Good that you spot this -- it makes me more disciplined in the future.